src/plat/polarfire/config.cmake

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright 2020, DornerWorks
+#
+# This software may be distributed and modified according to the terms of
+# the GNU General Public License version 2. Note that NO WARRANTY is provided.
+# See "LICENSE_GPLv2.txt" for details.
+#
```

Comment:
This part is no longer required with SPDX tags. For consistency with the rest of the headers I'd suggest removing it.